### ISSUE TITLE
Image labelling not working on iOS

### ISSUE THREAD

**Issue by mrados7** [mrousavy/vision-camera-image-labeller#12]
So I tested example app on iOS 15 iPhone 12 and it's not working properly. 
Results are always same:
* sky
*  pattern
*  curtain

Results have nothing to do with what's in the camera frame. 
From logs I get this in xcode:
```[WARNING]The specified colorspace format is not supported. Falling back on Libyuv.```

**Comment by valstu** [comment]
Noticed the same issue with iPhone 12 Pro, did you figure this out?

**Comment by mrados7** [comment]
Nope, unfortunately.

**Comment by valstu** [comment]
One thing I figured out is that if you set `preset="cif-352x288"` for <Camera /> it gives other results but I think it should work with better quality 🤔 will continue to debug.

**Comment by mrousavy** [comment]
I think this is because iOS has new 10-bit x420 formats that are not supported by the image labeller. Maybe we can upgrade the native library to support it, or choose a format that's not x420. Thomas is working on a PR in VisionCamera to expose that prop in format.

**Comment by valstu** [comment]
Yeah, I came to same conclusion that this is related to this. Here's the PR @mrousavy mentioned. https://github.com/mrousavy/react-native-vision-camera/pull/559 and it will hopefully solve this. My hunch is that MLKit is now receiving "images" something like in this Stack Overflow post https://stackoverflow.com/questions/42997462/convert-cmsamplebuffer-to-uiimage and that's why it is always returning `pattern, curtain, sky, monochrome` results. 🤷‍♂️

**Comment by willcray** [comment]
I'm still receiving this issue. I upgraded to RNVC 2.12.0, which should include [mrousavy/react-native-vision-camera#559](https://github.com/mrousavy/react-native-vision-camera/pull/559). My `pixelFormat` is set to `420v`, but the issue remains.

**Comment by kirill3333** [comment]
Any updates on it @mrousavy

**Comment by ademirmochi** [comment]
Is there any solution to this issue?

**Comment by DeepikaSharma5** [comment]
@mrousavy Any fixes for the image detection on IOS

**Comment by ste00martin** [comment]
bump
